### PR TITLE
Enable passed v8 tests

### PR DESCRIPTION
**Pull request by rerobika**
Signed-off-by: Robert Fancsik frobert@inf.u-szeged.hu

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ matrix:
         - ninja -Cout/linux/x64/release
         - cp ./out/linux/x64/release/escargot ./escargot
         - travis_wait 30 tools/run-tests.py --arch=x86_64 jetstream-only-simple chakracore
-        - tools/run-tests.py --arch=x86_64 jetstream-only-cdjs sunspider-js internal octane jsc-stress v8 spidermonkey regression-tests es2015 intl
+        - travis_wait 30 tools/run-tests.py --arch=x86_64 jetstream-only-cdjs sunspider-js internal octane jsc-stress v8 spidermonkey regression-tests es2015 intl
```
Comment (clover2123) [inline .travis.yml]: Do these tests require a longer running time than before?

File: .travis.yml [hunk 2]
```diff
@@ -77,7 +77,7 @@ matrix:
         - ninja -Cout/linux/x86/release
         - cp ./out/linux/x86/release/escargot ./escargot
         - travis_wait 30 tools/run-tests.py --arch=x86 jetstream-only-simple chakracore
-        - tools/run-tests.py --arch=x86 jetstream-only-cdjs sunspider-js internal octane jsc-stress v8 spidermonkey regression-tests es2015 intl test262
+        - travis_wait 30 tools/run-tests.py --arch=x86 jetstream-only-cdjs sunspider-js internal octane jsc-stress v8 spidermonkey regression-tests es2015 intl test262
         - tools/run-tests.py --arch=x86 test262
```
Comment (clover2123) [inline .travis.yml]: Currently, `test262` is running twice (typo). Would you fix it to run just once? Thanks:)

File: .travis.yml (hunk 1 above)
Comment (rerobika) [reply .travis.yml]: Oops, I left it here. I think it van be removed.

File: .travis.yml (hunk 2 above)
Comment (rerobika) [reply .travis.yml]: Sure!

**Comment by clover2123** [comment]
After #313 was landed, it would pass more test cases. Did you rebase this patch and check the whole v8 tests? It would be great if you check once more. Thanks:)

File: .travis.yml (hunk 2 above)
Comment (clover2123) [reply .travis.yml]: This typo is fixed by #323. Please sync `.travis.yml` file to that of master branch.

**Comment by bbrto21** [comment]
@rerobika Please, rebase this PR to the master branch.

**Comment by rerobika** [comment]
Closing after #341.

**Pull request closed without merging**